csharp/src/Google.Protobuf/Collections/RepeatedFieldExtensions.cs

```diff
@@ -0,0 +1,53 @@
+#region Copyright notice and license
+// Protocol Buffers - Google's data interchange format
+// Copyright 2015 Google Inc.  All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+#endregion
+
+using System;
+
+namespace Google.Protobuf.Collections
```

Comment:
Should the namespace just be `Google.Protobuf`? It's rare that people will directly reference `Google.Protobuf.Collections` in their app because they're just modifying collections created by generated code.

Having the extension method in a rarely used namespace makes it hard to discover/use.

Reply:
Makes sense to me.